style.css

```diff
@@ -2,3 +2,218 @@
 /*font-family: 'Montserrat', sans-serif;
 font-family: 'Pacifico', cursive;
 font-family: 'Poppins', sans-serif;*/
+*,*:before, *:after {
+    padding: 0;
+    margin: 0;
+    border: 0;
+    box-sizing: border-box;
+}
+html, body {
+    height: 100%;
+    font-family: 'Poppins', sans-serif;
+}
+.container {
+    max-width: 1180px;
+    margin: 0px auto;
+    padding: 0px 10px;
+}
+.header {
+width: 100%;
+top: 0;
+left: 0;
+z-index: 50;
+position: fixed; 
+background-color: white;   
+}
+.header:before {
+    content: '';
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    position: absolute; 
+    background-color:white; 
+    z-index: 2;
+}
+.header_logo {
+    text-decoration: none;
+    flex: 0 0 60px;
+    position: relative;
+    z-index: 4;
+}
+.header_cart {
+    flex: 0 0 40px;
+    float: right;
+    z-index: 4;
+    display: flex;
+    border-right: 2px solid black;
+    margin-right: 15px;
+}
+.cart {
+    background-color: white;
+    width: 25px;
+    height: 25px;
+}
+.header_cart img {
+    width: 25px;
+    height: 25px;
+    padding-right: 5px;
+    background-color: white;
+}
+.header_cart h5 {
+    padding: 3px 10px 0px 5px;
+}
+#renome {
+    margin-left: 10px;
+    margin-bottom: 10px;
+    font-family: 'Pacifico', cursive;
+    font-weight: 200;
+    color:black;
+    z-index: 5;
+}
+.header_body {
+  position: relative; 
+  display: flex;
+  justify-content: space-between;
+  height: 80px;
+  align-items: center;
+}
+.container_menu {
+  position: relative; 
+  display: flex;
+}
+.header_burger {
+    display: none;
+}
+.header_menu {
+ 
+}
+.header_list {
+    display: flex;  
+    position: relative;
+    z-index: 2;
+    padding: 0px 15px 0px 0px;
+}
+.cart_list {
+    display: none;
+}
+.cart_list.active {
+    display: block;  
+    position: relative;
+    right: 50px;
+    z-index: 2;
+    padding: 20px 10px 10px 10px;
+    width: 100px;
+    height: 140px;
+    background-color: white;
+}
+.header_list li {
+    list-style: none;
+    margin: 0px 0px 0px 10px;
```

Comment:
If it's 0, then it's 0. No need to write px after it.

Reply:
Fixed it (in all places)